### PR TITLE
Update to Fury 2.3

### LINKED ISSUE #6: Tests Fail on a Fresh Clone
I cloned the project and attempted to run the tests and ran into the following errors:

```
> fury-adapter-apib-serializer@0.1.2 test /Users/Peter/Code/localmed/fury-adapter-apib-serializer
> peasant test

🔎  Running tests...

   API Blueprint serializer adapter
     1) serializes basic.json
     2) serializes sample.json

   Executed 2 tests in 1 suites in 225ms
       0 passes
       2 failed

   1) API Blueprint serializer adapter serializes basic.json

       (/Users/Peter/Code/localmed/fury-adapter-apib-serializer/template.nunjucks) [Line 97, Column 20]
         TypeError: headers.include is not a function

       at Object.exports.prettifyError (node_modules/nunjucks/src/lib.js:34:15)

       33 |         var old = err;
       34 |         err = new Error(old.message);
       35 |         err.name = old.name;

       at node_modules/nunjucks/src/environment.js:486:31

       485 |                 if(err) {
       486 |                     err = lib.prettifyError(_this.path, _this.env.opts.dev, err);
       487 |                 }

       at new_cls.root [as rootRenderFunc] (eval at _compile (node_modules/nunjucks/src/environment.js:565:24), <anonymous>:611:3)
       at new_cls.render (node_modules/nunjucks/src/environment.js:479:15)

       479 |         _this.rootRenderFunc(
       480 |             _this.env,

       at node_modules/nunjucks/src/environment.js:312:35

       311 |             else {
       312 |                 syncResult = tmpl.render(ctx, cb);
       313 |             }

       at createTemplate (node_modules/nunjucks/src/environment.js:234:25)

       233 |                     if(cb) {
       234 |                         cb(null, tmpl);
       235 |                     }

       at handle (node_modules/nunjucks/src/environment.js:249:25)

       248 |                         src.loader = loader;
       249 |                         done(null, src);
       250 |                     }

       at node_modules/nunjucks/src/environment.js:263:21

       262 |                 else {
       263 |                     handle(null, loader.getSource(name));
       264 |                 }

       at next (node_modules/nunjucks/src/lib.js:207:13)

       206 |         if(i < arr.length) {
       207 |             iter(arr[i], i, next, cb);
       208 |         }

       at Object.exports.asyncIter (node_modules/nunjucks/src/lib.js:214:5)

       214 |     next();
       215 | };


   2) API Blueprint serializer adapter serializes sample.json

       (/Users/Peter/Code/localmed/fury-adapter-apib-serializer/template.nunjucks) [Line 51, Column 23]
         Error: Unable to call `item["key"]["toValue"]`, which is undefined or falsey

       at Object.exports.prettifyError (node_modules/nunjucks/src/lib.js:34:15)

       33 |         var old = err;
       34 |         err = new Error(old.message);
       35 |         err.name = old.name;

       at node_modules/nunjucks/src/environment.js:486:31

       485 |                 if(err) {
       486 |                     err = lib.prettifyError(_this.path, _this.env.opts.dev, err);
       487 |                 }

       at new_cls.root [as rootRenderFunc] (eval at _compile (node_modules/nunjucks/src/environment.js:565:24), <anonymous>:611:3)
       at new_cls.render (node_modules/nunjucks/src/environment.js:479:15)

       479 |         _this.rootRenderFunc(
       480 |             _this.env,

       at node_modules/nunjucks/src/environment.js:312:35

       311 |             else {
       312 |                 syncResult = tmpl.render(ctx, cb);
       313 |             }

       at new_cls.getTemplate (node_modules/nunjucks/src/environment.js:194:17)

       193 |             if(cb) {
       194 |                 cb(null, tmpl);
       195 |             }

       at new_cls.render (node_modules/nunjucks/src/environment.js:304:14)

       304 |         this.getTemplate(name, function(err, tmpl) {
       305 |             if(err && cb) {

       at Object.module.exports.render (node_modules/nunjucks/index.js:72:14)

       72 |     return e.render(name, ctx, cb);
       73 | };

       at Object.serialize (src/adapter.js:31:12)

       30 | export function serialize({api}, done) {
       31 |   nunjucks.render('template.nunjucks', {api}, (err, apib) => {
       32 |     if (err) {

       at Context.<anonymous> (test/adapter.js:34:15)

       34 |       adapter.serialize({api: refract}, (serializeErr, serialized) => {
       35 |         if (serializeErr) { return done(serializeErr); }


npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/Peter/.nvm/versions/node/v6.6.0/bin/node" "/Users/Peter/.nvm/versions/node/v6.6.0/bin/npm" "run" "test"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! fury-adapter-apib-serializer@0.1.2 test: `peasant test`
npm ERR! Exit status 1
```

### PR DESCRIPTION
**Pull request by kylef**
Not yet ready for review since it's still failing. Seems the template is more broken than I originally though 😞.

Fixes #6.

### CODE REVIEW

**Comment by pksunkara** [comment]
@kylef It's only a linting error.

**Review by pksunkara — DISMISSED** [review]

**Comment by kylef** [comment]
> @kylef It's only a linting error.

That was certainly not true, there were many test failures too. I've resolved them all, seems to be related to how API Elements has evolved and certain piece of information were moved. I've written further description on the commits, I'd suggest reviewing per commit.

File: src/mson.js [hunk 1]
```diff
@@ -202,13 +202,23 @@ function handle(name, element, {parent = null, spaces = 4, marker = '+',
...
-  const title = mson.title;
-
-  return handle(title || 'Attributes', mson, {
-    initialMarker: title ? '###' : '+',
-    initialIndent: title ? false : true,
+export function renderDataStructure(dataStructure) {
+  const mson = dataStructure.content[0];
+  const title = mson.id;
+
+  return handle(title, mson, {
```
Comment (pksunkara) [inline src/mson.js:209]: Why not use `mson.id` directly instead of `title` in this line?

File: template.nunjucks [hunk 2]
```diff
@@ -151,7 +151,7 @@ FORMAT: 1A
 ## Data Structures
 
 {% for dataStructure in dataStructures.content %}
-{{ dataStructure.content|mson }}
+{{ dataStructure|dataStructure }}
```
Comment (pksunkara) [inline template.nunjucks:154]: I don't understand this change.

**Review by pksunkara — COMMENTED** [review]
Few comments, otherwise, lgtm.

File: template.nunjucks (hunk 2 above)
Comment (kylef) [reply template.nunjucks:154]: Resource data structures (example: https://github.com/apiaryio/fury-adapter-apib-serializer/blob/kylef/update-fury/test/fixtures/sample.json#L97) also contain an `id` so the existing code saw `id` on dataStructure element and assumed it was a standalone `## Data Structure` and would render it as such incorrectly.

I split out the existing code into two separate filters (one still called mson) and another called `dataStructure` as I felt this was a better name in the content of rendering a data structure. This allows `dataStructure` filter to use `##` headings and the other `mson` (attributes) which render the attributes as list items.

**Comment by pksunkara** [comment]
Please tag and release.

**Pull request merged**